lib/ClangImporter/ClangImporter.cpp

```diff
@@ -781,6 +781,11 @@ importer::addCommonInvocationArguments(
     invocationArgStrs.push_back("-march=z13");
   }
 
+  if (triple.isOSLinux() && triple.getArch() == llvm::Triple::x86_64) {
```

Comment:
Any reason to not unconditionally do this?  Does this cause problems on macOS?  I think that we should be doing this on WIndows, Linux, and FreeBSD at least.

Reply:
Good point; it's already present on Darwin targets, but it doesn't hurt to add it anyway. Updating.